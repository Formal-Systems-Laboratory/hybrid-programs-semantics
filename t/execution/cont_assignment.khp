vars x, f, t, ep ; ep := 5.0 ; f := * ; t := 0.0 ; x ' = f , t' = 1.0 & x >= 0.0 , t <= ep
