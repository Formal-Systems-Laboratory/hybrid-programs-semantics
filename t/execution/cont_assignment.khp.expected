<generatedTop>
  <k>
    .
  </k>
  <state>
    ep |-> 5.0
    f |-> #VarReal ( f0 )
    t |-> 0.0 + 1.0 * #VarReal ( tpost )
    t_traj |-> #interval { #quantified ( #VarReal ( tpost2 ) ) , ( 0.0 <= #VarReal ( tpost2 ) ) && #VarReal ( tpost2 ) <= #VarReal ( tpost ) } ( 0.0 + 1.0 * #VarReal ( tpost2 ) )
    x |-> #VarReal ( x ) + #VarReal ( f0 ) * #VarReal ( tpost )
    x_traj |-> #interval { #quantified ( #VarReal ( tpost1 ) ) , ( 0.0 <= #VarReal ( tpost1 ) ) && #VarReal ( tpost1 ) <= #VarReal ( tpost ) } ( #VarReal ( x ) + #VarReal ( f0 ) * #VarReal ( tpost1 ) )
  </state>
  <gPgmVars>
    ep , t , f , x , .Ids
  </gPgmVars>
  <gNonDetAssignments>
    f |-> #VarReal ( f0 )
  </gNonDetAssignments>
  <gEvolutionConditions>
    SetItem ( #VarReal ( tpost ) >= 0.0 )
    SetItem ( #interval { #quantified ( #VarReal ( tpost1 ) ) , ( 0.0 <= #VarReal ( tpost1 ) ) && #VarReal ( tpost1 ) <= #VarReal ( tpost ) } ( #VarReal ( x ) + #VarReal ( f0 ) * #VarReal ( tpost1 ) ) >=Trajectory 0.0 )
    SetItem ( #interval { #quantified ( #VarReal ( tpost2 ) ) , ( 0.0 <= #VarReal ( tpost2 ) ) && #VarReal ( tpost2 ) <= #VarReal ( tpost ) } ( 0.0 + 1.0 * #VarReal ( tpost2 ) ) <=Trajectory 5.0 )
  </gEvolutionConditions>
  <counter>
    3
  </counter>
  .SynthesisCell
</generatedTop>
